gson/src/main/java/com/google/gson/internal/JavaVersion.java

```diff
@@ -37,7 +37,7 @@ static int getMajorJavaVersion(String javaVersion) {
       version = extractBeginningInt(javaVersion);
     }
     if (version == -1) {
-      return 6;  // Choose minimum supported JDK version as default
+      return 7;  // Choose minimum supported JDK version as default
```

Comment:
Missed that for #2043, but this change currently has no effect anyway because Gson only uses `JavaVersion.isJava9OrLater()`.